### PR TITLE
DROOLS-2689 : The ability to hide a Simulation Scenario asset based on a flag

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2689

I guess we can make a bundle out of these:
* https://github.com/kiegroup/kie-wb-common/pull/2126
* https://github.com/kiegroup/drools-wb/pull/934
* https://github.com/kiegroup/kie-wb-distributions/pull/813

### CODE REVIEW

**Comment by manstis** [comment]
@Rikkola IDK who, from QE, should be reviewing the new Test Scenario Editor?

**Comment by Rikkola** [comment]
@manstis @kkufova

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/resources/org/kie/workbench/common/workbench/client/resources/i18n/DefaultWorkbenchConstants.properties [hunk 1]
```diff
@@ -83,4 +83,5 @@ InvalidBusResponseProbablySessionTimeout=Invalid response received from the serv
 LayoutEditorComponentPalette=Components
 ErrorDetailsSuccessfullyCopiedToClipboard=Error details were copied to clipboard.
 ErrorDetailsFailedToBeCopiedToClipboard=Error details could not be copied to the clipboard because this browser does not support it.
+ScenarioSimulationEditor=Test Scenario Editor
```
Comment (kkufova) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/resources/org/kie/workbench/common/workbench/client/resources/i18n/DefaultWorkbenchConstants.properties]: According to [this document](https://docs.google.com/presentation/d/1uBgHnBOzmsXvRUs7fFToLsjVLs3ag6lCnc6XFVUk9Ao/edit?ts=5b96322e#slide=id.g3efa8f3c6c_7_130), the new editor is supposed to be called **Test Scenarios (Preview)**. Could you please rename it?

Comment (Rikkola) [reply kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/resources/org/kie/workbench/common/workbench/client/resources/i18n/DefaultWorkbenchConstants.properties]: Sure thing.

**Comment by Rikkola** [comment]
@kkufova Updated. The problem with the name was that nobody could not come up with a better name and in the end the editor is the same old thing with a different layout.

**Comment by Rikkola** [comment]
@manstis I think we can merge this one and leave the kie-wb-distributions PR out for now.

**Pull request merged**